### DISCUSSION TITLE
Import Pole FIgure

### DISCUSSION THREAD

**Discussion by Andre-calazans** [mtex-toolbox/mtex#545] [Ask Anything] [answered]
Hi all,

I am beginner level user and I am trying to import a pole figure through the import_wizard. My data is only 3 columns (polar Angle/Polar angle/intensity). However, the pole figure generic interface window do not allow me to press finish. Does anyone know what is happening?

[Presentation3.pptx](https://github.com/mtex-toolbox/mtex/files/5437192/Presentation3.pptx)

**Comment by ralfHielscher** [comment, marked as answer]
You should select for the second angle `Azimuth Angle`

**Reply by Andre-calazans** [reply]
Hi Ralf,

Thank you for you feedback!

I have already tried Azimuth angle for the first 2 columns, but the finish option is not allowed.

Note, I collected my CPO from neutron diffraction, but I have already converted to pole figures. Now I want to import these pole figures to calculate my seismic velocities.

Question:

I have several CPO from each mineral phase. Should I superpose them to calculate the velocity?


Cheers,
Andre

Cheers,
Andre

**Reply by ralfHielscher** [reply]
You need to select 

`polar angle` for the first column and `azimuth angle` for the second column.

Ralf.

**Reply by Andre-calazans** [reply]
Thank you

**Comment by kilir** [comment]
Hi,
a) maybe you can share an example file
b) if you compute an odf for each phase, you can can get numbers for bulk velocities in several ways. See here: https://mtex-toolbox.github.io/CPOSeismicProperties.html#13
Cheers,
Rüdiger

**Reply by Andre-calazans** [reply]
Hi Rudiger,

Thank you
Andre
